ocaml/parsing/ast_iterator.ml

```diff
@@ -29,6 +29,7 @@ type iterator = {
   attributes: iterator -> attribute list -> unit;
   binding_op: iterator -> binding_op -> unit;
   case: iterator -> case -> unit;
+  case_rhs: iterator -> case_rhs -> unit;
```

Comment:
Nit: consider putting below `cases`? I know this isn't literally alphabetical order, but it feels better.

Reply:
Done, and ditto for ast_mapper